app/src/main/java/com/github/gotify/messages/MessagesActivity.java

```diff
@@ -357,11 +362,26 @@ protected void onDestroy() {
     }
 
     private class SwipeToDeleteCallback extends ItemTouchHelper.SimpleCallback {
-        ListMessageAdapter adapter;
+        private ListMessageAdapter adapter;
+        private Drawable icon;
+        private final ColorDrawable background;
 
         public SwipeToDeleteCallback(ListMessageAdapter adapter) {
             super(0, ItemTouchHelper.LEFT | ItemTouchHelper.RIGHT);
             this.adapter = adapter;
+
+            int backgroundColorId =
+                    ContextCompat.getColor(MessagesActivity.this, R.color.swipeBackground);
+            int iconColorId = ContextCompat.getColor(MessagesActivity.this, R.color.swipeIcon);
+
+            Drawable drawable =
+                    ContextCompat.getDrawable(MessagesActivity.this, R.drawable.ic_delete);
+            if (drawable != null) {
```

Comment:
Can this be null? If that's the case, then icon.draw(c); in onChildDraw must get a nullcheck too.

Reply:
True